starknet_py/tests/e2e/account/account_client_test.py

```diff
@@ -142,23 +144,40 @@ async def test_create_account_client_with_signer(run_devnet):
 
 
 @pytest.mark.asyncio
-async def test_sending_multicall(account_client, map_contract):
-    calls = [
-        map_contract.functions["put"].prepare(key=10, value=10),
-        map_contract.functions["put"].prepare(key=20, value=20),
-    ]
+async def test_sending_multicall(account_clients, map_contract):
+    for account_client in account_clients:
+        calls = [
+            map_contract.functions["put"].prepare(key=10, value=10),
+            map_contract.functions["put"].prepare(
+                key=account_client.signer.public_key, value=account_client.address
+            ),
+        ]
 
-    res = await account_client.execute(calls, int(1e20))
-    await account_client.wait_for_tx(res.hash)
+        res = await account_client.execute(calls, int(1e20))
+        await account_client.wait_for_tx(res.transaction_hash)
 
-    (value,) = await map_contract.functions["get"].call(key=20)
+        (value,) = await map_contract.functions["get"].call(
+            key=account_client.signer.public_key
```

Comment:
i don't get it. why do we use that now?

Reply:
it was supposed to be a simple k-v and now we store account client's address?

Reply:
can't we somehow parametrize it? 
i.e: 

`for client, (k, v)  in zip(clients, ((k1,v1),(k2,v2)))`

Reply:
Right, we can. I'll fix it